src/group/types.ts

```diff
@@ -0,0 +1,3 @@
+import { User } from '../user/types';
+
+export type GroupMember = User;
```

Comment:
I don't think it's needed to make a new type for this. I actually think it's kinda helpful to know that the `groups` service deals with users, not a special `GroupMember`.

Reply:
So would I just import the User from user/type.ts file and use the user instead?